clients/geth/specular/Makefile

```diff
@@ -20,7 +20,7 @@ clef:
 	@echo "Done building clef."
 	@echo "Run \"$(GOBIN)/clef\" to launch clef."
 
-geth-docker:
+geth-docker: bindings
```

Comment:
i think `geth-docker` and `geth` are equivalent now so we can just get rid of the former right